content/en/docs/tasks/debug-application-cluster/debug-running-pod.md

```diff
@@ -99,7 +99,7 @@ kubectl run ephemeral-demo --image=k8s.gcr.io/pause:3.1 --restart=Never
 ```
 
 The examples in this section use the `pause` container image because it does not
-contain userland debugging utilities, but this method works with all container
+contain user and debugging utilities, but this method works with all container
```

Comment:
👋 @juampynr . Thanks for your contribution!
This change seems okay, though I am not clear on the meaning of "user and debugging utilities".
What do you think about changing the text to:
... contain debugging utilities, but this ...
or
... contain user debugging utilities, but this ...

Reply:
Hi @kbhawkey !

`contain debugging utilities, but this..` Sounds good to me. Changing...

Reply:
👍